### DISCUSSION TITLE
Change user agent?

### DISCUSSION THREAD

**Discussion by Joaqstarr** [Strip3s/PhoenixBot#77] [Q&A]
Kinda new to this and figuring it out. Target isn't letting the bot sign in. Does that have anything to do with user agents? If so how would I go about changing or fixing the issue?

**Comment by infowire** [comment]
You can login manually when the browser logs in for you and fails, duplicate tab and log in once the refreshing page goes to you product. I made an issue you can look. Hopefully it works.
